addon/-private/decorators/localized-attr.js

```diff
@@ -11,7 +12,23 @@ export default function (...args) {
   attrComputed.get = function () {
     const value = getter.call(this);
 
-    return value && value[this.getFieldLocale()];
+    if (!value) {
+      return;
+    }
+
+    const {
+      localizedModel: { fallbacks = [] } = {}
+    } = getOwner(this).resolveRegistration('config:environment');
+
+    const keys = [this.getFieldLocale(), ...fallbacks];
+    const key = (
+      // Get the first valid value from the selected keys.
+      keys.find(key => Boolean(value[key])) ||
+      // If that doesn't yield a value get the first of any key.
+      Object.keys(value).find(key => Boolean(value[key]))
+    );
```

Comment:
This should only be done if the `fallbacks` array has actually content. Since apps which are not using this will now always get the first truthy value from the languages if i understand this correctly. But this is not expected in some other apps that use `ember-localized-model`. Some apps actually expect the field to be empty if there is no content in the specified language.

I would prefer that these would be two configurations `allowFallback` and `fallbackOrder`. One enables that just any truthy value is returned, and the other then specifies an order in which to check.

Reply:
Otherwise i think this would be a breaking change.

Reply:
Or maybe `allowFallback: Boolean | [LanguageKey]` so we have only one config field.

Reply:
What do you think about this?

Reply:
I'll update :smiley:

Reply:
Sorry. I didn't see that you wrote more than one comment...